SpecFlowSolar/Features/CartTests.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: CartTests
+@mytag
+Scenario: add and delete product and return to home
+	Given open home page in CartTests
```

Comment:
this is business language which should be clear for any stakeholder. There should not be "in CartTests"